allauth/socialaccount/providers/eventbrite/provider.py

```diff
@@ -11,6 +15,14 @@ def get_avatar_url(self):
         """Return avatar url."""
         return self.account.extra_data['image_id']
 
+    def to_str(self):
+        """Return string of account name."""
+        return self.account.extra_data['name']
+
+    def get_profile_url(self):
```

Comment:
You can keep it, but then you should return the end-user URL linking to the user profile. Now you are returning the API URL endpoint.